.vscode/settings.json

```diff
@@ -0,0 +1,24 @@
+{
+  "editor.tabSize": 2,
+  "editor.detectIndentation": false,
+  "search.exclude": {
+    "package-lock.json": true
+  },
+  "editor.defaultFormatter": "dbaeumer.vscode-eslint",
+  "editor.formatOnSave": true,
```

Comment:
It should be on false, eslint doesn't actually format the code, but runs a quick fix on the save action. At the moment, if you have any code formatter configured in vscode, it will do both, quick fix and format it without taking eslint's configuration into account.